tests/scripts/thread-cert/node.py

```diff
@@ -419,6 +419,12 @@ def _process_packet_counters(self, counter):
     def nat64_set_enabled(self, enable):
         return self.call_dbus_method('io.openthread.BorderRouter', 'SetNat64Enabled', enable)
 
+    def start_ephemeral_key_mode(self, value):
```

Comment:
It may not be clear enough what 'value' means here. Please rename it.

Reply:
Accept, thanks!